pkgs/tools/networking/linux-router/default.nix

```diff
@@ -0,0 +1,89 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
+
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+
+# Dependencies
```

Comment:
```suggestion
# Dependencies
```

Reply:
I'm not quite sure what you changed here. Maybe GitHub is hidings something for me :confused:

Reply:
Sorry, I meant remove the comment and the blank here.

Reply:
I have since merged the sections `Dependencies` and `General dependencies`. Then there is also the section `Wifi hotspot dependencies`. Is this okay or do you think I should complete remove those "headings"?

Reply:
I like the idea of separating the imported packages by what the program/runtime dependencies and what the build dependencies are. Also the author of `linux-router` is listing the wifi hotspot dependencies separately, maybe because they can be considered optional, when you don't have Wifi. Would it make sense to add `useWifiDependencies = true;` and then have another package like:

```nix
  linux-router = callPackage ../tools/networking/linux-router { useWifiDependencies = false; };
```

Reply:
Your sorting of the packages looks good now. Maybe you could get rid of some blank lines to make it more compact. The comments Dependencies and General dependencies are not necessary IMHO.

Beyond the grouping of features and packages there is no clear guideline of how the inputs should be sorted. A common pattern is that `iib`, `stdenv` etc. are named first and very often you will find packages group that will go into `nativeBuildInputs` and `buildInputs` respectively.

Reply:
Okay, I have removed the blank lines.

What do you think about a "no-wifi" version of the package for let's say servers that have no wifi at all? Is it better to make each wifi dependency optional with a `use*` variable and disable them all for the "no-wifi" version or is it better to have a separate `useWifi` variable which disables all of the wifi dependencies at once?